src/templates/disease-cell-line.tsx

```diff
@@ -1,60 +1,221 @@
+import React, { useState } from "react";
+import { graphql, Link } from "gatsby";
+import { Button, Card, Divider, Flex, Tooltip } from "antd";
+
 import Layout from "../components/Layout";
+import {
+    Clone,
+    DiseaseCellLineFrontmatter,
+    GeneFrontMatter,
+} from "../component-queries/types";
+import { formatCellLineId, getCloneSummary } from "../utils";
+import CloneTable from "../components/CloneTable";
+import Icon from "@ant-design/icons";
+import InfoPanel from "../components/shared/InfoPanel";
+import { primaryColor } from "../style/theme";
+import {
+    DarkThemeGhostButton,
+    DefaultButton,
+} from "../components/shared/Buttons";
+
+const {
+    container,
+    title,
+    extraLargeButton,
+} = require("../style/main-card.module.css");
+const Share = require("../img/share-icon.svg");
+const Arrow = require("../img/arrow.svg");
 
+interface DiseaseCellLineTemplateProps {
+    href: string;
+    cellLineId: string;
+    geneName: string;
+    geneSymbol: string;
+    status: string;
+    snp: string;
+    orderLink: string;
+    certificateOfAnalysis: string;
+    parentalLine: any;
+    parentLineGene: GeneFrontMatter;
+    clones: Clone[];
+    healthCertificate: string;
+}
 // eslint-disable-next-line
-export const CellLineTemplate = ({
+export const DiseaseCellLineTemplate = ({
+    href,
     cellLineId,
-    cloneNumber,
-    gene,
-    tagLocation,
-    status,
-}) => {
-    return (
-        <section className="section">
-            <div className="container content">
-                <div className="columns">
-                    <div className="column is-10 is-offset-1">
-                        <h1 className="title is-size-2 has-text-weight-bold is-bold-light">
-                            AICS-{cellLineId}
-                        </h1>
-                        <p>Gene: {gene}</p>
-                    </div>
-                </div>
-            </div>
-        </section>
+    parentLineGene,
+    geneName,
+    geneSymbol,
+    clones,
+    snp,
+    orderLink,
+    certificateOfAnalysis,
+    parentalLine,
+    healthCertificate,
+}: DiseaseCellLineTemplateProps) => {
+    const defaultToolTipText = "Copy cell line link to clipboard";
+    const [toolTipText, setToolTipText] = useState(defaultToolTipText);
+    const tableData = [
+        {
+            key: "1",
+            label: "SNP",
+            children: snp,
+        },
+        {
+            key: "2",
+            label: "Gene Symbol",
+            children: geneSymbol,
+        },
+        {
+            key: "3",
+            label: "Gene Name",
+            children: geneName,
+        },
+        {
+            key: "4",
+            label: "Parental Line",
+            children: `${formatCellLineId(parentalLine.cell_line_id)} cl. ${
+                parentalLine.clone_number
+            } ${parentLineGene.symbol}`,
+        },
+    ];
+    const urlToCopy = `Check out this cell line from the Allen Cell Collection! ${href}`;
```

Comment:
I feel like I usually only see this pattern when sharing is done on mobile devices via the native sharing API between apps... as a user I'd expect to only get the URL, and not the additional text. Worth discussing with UX if this didn't come directly from them?